src/libraries/System.Security.Cryptography/tests/SlhDsaOpenSslConstructionTests.Unix.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Security.Cryptography.SLHDsa.Tests
+{
+    [ConditionalClass(typeof(SlhDsa), nameof(SlhDsa.IsSupported))]
+    public class SlhDsaOpenSslConstructionTests : SlhDsaConstructionTestsBase
```

Comment:
There are positive tests for when SLH-DSA is supported, and negative tests for non-Unix.  We should have some sort of test that shows that the ctors don't AV or whatnot in the case where it's Unix on an older OSSL (or one with SLH-DSA disabled)

Reply:
Do we have SKUs that will intentionally not have new bits (i.e. OpenSSL < 3.5)? Or will we need to simulate this in tests by disabling SLH-DSA somehow? I know the CI machines don't support it at the moment, but I assume eventually they would all do so and we'd need to make sure your suggested test still tests old OpenSSL/disabled SLH-DSA.

Reply:
>  I know the CI machines don't support it at the moment, but I assume eventually they would all do 

It will be many years before OpenSSL 3.5 is ubiquitous. I would expect we continue to have at least one pipeline to be on < OpenSSL 3.5 for a long time. The unfortunate reality of OpenSSL for .NET is that there are a good number of configurations that simply aren't exercised in CI.

Having a unit test is good though, even if it will only be practically run for a few years.

Reply:
Removed the condition so the tests will run for Unix OpenSSL < 3.5 as well.